generators/client/templates/angular/src/main/webapp/_index.html

```diff
@@ -41,6 +43,11 @@
             script.type = 'text/javascript';
             document.getElementsByTagName('head')[0].appendChild(script)
         }
+        /*if ('serviceWorker' in navigator) {
```

Comment:
There is no harm in uncommenting it 🤔, but really give this options to users such that they can use this, if they want to...